packages/modules/src/modules/banners/common/choiceCard/ChoiceCardInteractive.tsx

```diff
@@ -116,6 +147,30 @@ const buildStyles = (design: ChoiceCardSettings | undefined, frequencyColumns: n
             ${until.mobileMedium} {
                 font-size: 10px;
             }
+
+            & + label {
+                ${buttonTextColour ? `color: ${buttonTextColour};` : ''}
+                ${buttonColour ? `background-color: ${buttonColour};` : ''}
+                ${buttonBorderColour ? `box-shadow: inset 0 0 0 2px ${buttonBorderColour};` : ''}
+            }
+
+            &:hover + label {
+                ${buttonTextColour ? `color: ${buttonTextColour};` : ''}
+                ${buttonColour ? `background-color: ${buttonColour};` : ''}
+                ${buttonBorderColour
+                    ? `box-shadow: inset 0 0 0 2px ${buttonSelectBorderColour};`
```

Comment:
I think you've used the wrong variable in a few places here!

Reply:
Ah ... we lost the border width expansion on the last change. Fixed.

Reply:
I don't understand why you're using a different variable to the one you're checking here

Reply:
Fixed again!